ansible/vars/all.yml

```diff
@@ -5,8 +5,8 @@ rhel_pool_id:
 manage_repos: false
 
 setup_microshift_host: false
-prometheus_logging: true
-install_microshift: false
+prometheus_logging: "{{ prometheus_logging_arg | default('true')  }}"
```

Comment:
I think it should just be `prometheus_logging: "{{ prometheus_logging | default('true')  }}"`
Referencing the same variable.

Reply:
I tried that before. Unfortunately, it did not work to use the same name. It breaks somewhere down the execution.